### PR TITLE
Simulation study plots

### PR DESCRIPTION
**Pull request by zakvarty**
Add plots of case study probability density functions for the supplementary material. 

CM to check beta distribution used, suggesting beta(1,2) as alternative to beta(1, 0.5)

### CODE REVIEW

File: simulation_study/plot-case-study-denisties.R [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+  gpd_scale = 0.5
+  gpd_shape = 0.1
+  
+  prop_to_density <- function(x, a = alpha, b = beta){
+    dgpd(x, shape = 0.1, scale = 0.5) * pbeta(x, a, b)
+  } 
+
+  # Area correction to get valid pdf
+    AUC_0_to_1 <- integrate(prop_to_density, 0, 1)$value
+    AUC_1_to_inf <- 1 - pgpd(q = 1,shape = gpd_shape, scale = gpd_scale)
```
Comment (conor-murphy4) [inline simulation_study/plot-case-study-denisties.R:37]: I think AUC_1_to_inf should be using the adjusted GPD parameters conditioned on being above 1, i.e., scale=0.6?

File: simulation_study/plot-case-study-denisties.R [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  density[is_bgpd] <- dgpd(x[is_bgpd], shape = 0.1, scale = 0.5) *
+    pbeta(x[is_bgpd], shape1 = alpha, shape2 = beta)
+
+  ## Calculate area under the curve between 0 and 1
+  n_eval_points = 10001
+  eval_points <- ppoints(n_eval_points)
+  eval_values <- dgpd(eval_points, shape = 0.1, scale = 0.5) *
+    pbeta(eval_points, shape1 = alpha, shape2 = beta)
+
+  q <- mean(c(eval_values[-1], eval_values[-n_eval_points]))  # AUC from 0 to 1
```
Comment (conor-murphy4) [inline simulation_study/plot-case-study-denisties.R]: Not sure I understand this step. Why is the eval_values vector repeated within the mean calculation? And why is the first and last element removed?

**Review by conor-murphy4 — COMMENTED** [review]
Just a couple of comments to address.

File: simulation_study/plot-case-study-denisties.R (hunk 2 above)
Comment (zakvarty) [reply simulation_study/plot-case-study-denisties.R]: This was Reimann sum (i.e. midpoint heights * interval width). Have swapped for `integrate()` for clarity.

File: simulation_study/plot-case-study-denisties.R (hunk 1 above)
Comment (zakvarty) [reply simulation_study/plot-case-study-denisties.R:37]: I don't think so, can discuss tomorrow.

Comment (zakvarty) [reply simulation_study/plot-case-study-denisties.R:37]: Have added additonal plots comparing sample CDF to analytical CDFs using mine and Jon's expression.

**Pull request merged**